<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title>Error</title>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
</head>
<body>

<div class="app-content pt-4">
    <main class="container">
        <h1>Exception: {{ exception.getMessage() }}</h1>

        <p>Code: {{ exception.getCode() }}</p>
        <p>{{ exception.getFile() }} on line {{ exception.getLine() }}</p>
        {% for trace in exception.getTrace() %}
            {% if isset(trace['file']) and empty(trace['file']) == false %}
                <p>{{ trace['file'] }} on line {{ trace['file'] }}</p>
            {% endif %}
        {% endfor %}
    </main>
</div>

</body>
</html>